frontend/src/containers/TeachersList/TeachersList.js

```diff
@@ -199,7 +201,7 @@ const TeacherList = (props) => {
                     >
                         {t(SEND_SCHEDULE_FOR_TEACHER)}
                     </Button>
-                    <>
+                    {openSelect && (
```

Comment:
Should we hide the select if it`s closed?

Reply:
Actually MultiSelect it's a modal window and inside window select